lblod/frontend-gelinkt-notuleren
# Update editor to `9.6.1`

### Overview
This PR updates the editor to version 9.6.1. 
This should not be a breaking release.

Note: This should probably be reviewed by someone else than me as I took this over while @elpoelma is away. - @piemonkey 

##### connected issues and PRs:
https://github.com/lblod/frontend-reglementaire-bijlage/pull/230

### How to test/reproduce
- Open up the application
- Ensure that the different editor instances throughout the application still work as expected

### Challenges/uncertainties
Note: this update does not yet remove usage of API deprecations introduced in 9.6.0. 
This PR is specifically made to ensure that the 9.6.0 works as expected and does not include any unexpected breaking changes.
This PR also does not include any new features.
~~Once version 9.6.0 has been released, this PR can be updated.~~

~~Note: the CI (specifically the install step) will fail for this PR, as the plugins package does not officially support version 9.6.0-next.0 (it only supports the `^9.0.0` range).~~

### Checks PR readiness
